config/packages/twig.yaml

```diff
@@ -0,0 +1,7 @@
+twig:
+    paths: ['%kernel.project_dir%/templates']
```

Comment:
As soon as https://github.com/symfony/symfony/pull/23339 to be merged, the default path to override bundles will be `templates/bundles/`, so what about omit (i.e. for templates) the `src/Resources/` path to override the `TwigBundle`'s templates? this configuration should enough to move them:
```yaml
twig:
    paths:
        templates: ~
        templates/bundles/Twig: Twig
```
Then, the error template can live there `templates/bundles/Twig/Exception/error.html.twig`. WDYT?